scipy/optimize/_linprog.py

```diff
@@ -505,6 +506,7 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
            x: array([10., -3.]) # may vary
 
     """
+
```

Comment:
Need to mention HiGHS in the linprog documentation.

Reply:
Can docstring from `highs_wrapper` be used/linked to?  Most of this is already written up there